doc/classes/NavigationMesh.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="NavigationMesh" inherits="Resource" is_experimental="true" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A mesh to approximate the walkable areas and obstacles.
+		A NavigationMesh is used in pathfinding to describe the traversable area that is safe from collision and other obstructions, assuming an agent's center position at zero radius.
```

Comment:
```suggestion
		A mesh that defines the areas traversable by navigation agents.
```
or maybe, if we want to be a bit more verbose, we could have something like
```suggestion
		A mesh that defines the areas traversable by navigation agents that are safe from obstructions.
```
It's a brief description.

Likewise in 2D.